libraries/AP_HAL_ChibiOS/AnalogIn.cpp

```diff
@@ -446,8 +456,8 @@ void AnalogIn::_timer_tick(void)
 
 #if HAL_WITH_IO_MCU
     // now handle special inputs from IOMCU
-    _servorail_voltage = iomcu.get_vservo();
-    _rssi_voltage = iomcu.get_vrssi();
+    _servorail_voltage = iomcu.get_vservo_adc_count() * VOLTAGE_SCALING * HAL_IOMCU_VSERVO_SCALAR;
```

Comment:
```suggestion
    _servorail_voltage = iomcu.get_vservo_adc_count() * (VOLTAGE_SCALING * HAL_IOMCU_VSERVO_SCALAR);
```

Group the constant multiplications so that we don't have to do 2 float multiplications on constant values.

Reply:
Fixed